iwyu_path_util.cc

```diff
@@ -173,7 +176,11 @@ string MakeAbsolutePath(const string& base_path, const string& relative_path) {
 
 string GetParentPath(const string& path) {
   llvm::StringRef parent = llvm::sys::path::parent_path(path);
-  return parent.str();
+  string result = parent.str();
+  // Ensure trailing slash
+  if(!result.empty() && result[result.size() - 1] != '/')
```

Comment:
Nope. I only want to add the slash, when the path is non-empty. So at least the empty check needs to stay. The 2nd part might be over-optimized... `result.back()` would be nicer, but that's C++11